index.html

```diff
@@ -1,217 +1,359 @@
+            <ul class="buttons">
+              <li>ruby on rails</li>
+              <li>Css</li>
+              <li>javascript</li>
+              <li>html</li>
+            </ul>
+            <div class="see-project">
+              <button>See project</button>
+            </div>
+          </div>
+          <div class="img-patholder">
+            <div class="img-box">
+              <img src="assets/images/placeholder-image.png" alt="project 1" />
+            </div>
+            <h3 class="title-project">Multi-Post Stories <br />Gain+Glory</h3>
+            <ul class="buttons">
+              <li>ruby on rails</li>
+              <li>Css</li>
+              <li>javascript</li>
+              <li>html</li>
+            </ul>
+            <div class="see-project">
+              <button>See project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section id="about" class="title about-me">
+        <h2 class="title">About Me</h2>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="get-my-resume">
+          <button>Get My Resume</button>
+        </div>
+      </section>
+      <section class="about__cards">
+        <div class="img-patholder language">
+          <div class="box-language">
+            <h2 class="title">Languages</h2>
+            <ul class="buttons button-white-bg">
+              <li>ruby on rails</li>
+              <li>Css</li>
+              <li>javascript</li>
+              <li>html</li>
+            </ul>
+          </div>
+        </div>
+        <div class="img-patholder FrameWorks">
+          <div class="box">
+            <h2 class="title">FrameWorks</h2>
+            <ul class="buttons2 button-white-bg">
+              <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>Rspec</li>
+            </ul>
+            <ul class="buttons3 button-white-bg">
+              <li>Capybara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+        </div>
+        <div class="img-patholder skills">
+          <div class="box">
+            <h2 class="title">Skills</h2>
+            <ul class="buttons button-white-bg">
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+            </ul>
+            <ul class="buttons3 button-white-bg">
+              <li>Termenal</li>
+            </ul>
+          </div>
+        </div>
+      </section>
     </div>
-    
-</body>
-
-</html>
\ No newline at end of file
+    <footer>
+      <div class="contact__container" id="contact">
+        <h2 class="title">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form

```

Comment:
- [ ] According to the project requirements, you need to use a CSS Flexbox property to layout the form elements, please add it.